scene/main/viewport.cpp

```diff
@@ -3718,6 +3728,13 @@ Viewport::Viewport() {
 	default_texture->proxy = RS::get_singleton()->texture_proxy_create(texture_rid);
 
 	//internal_listener_2d = SpatialSound2DServer::get_singleton()->listener_create();
+	audio_listener_2d = false;
+	transparent_bg = false;
+	parent = nullptr;
+	listener = nullptr;
+	camera_3d = nullptr;
+	camera_2d = nullptr;
+	override_canvas_transform = false;
```

Comment:
This popped up after the latest rebase, which raised an eyebrow. With the exception of camera_3d/2d, I don't think I touched anything here. Which implies that it was there when I originally branched, removed, and then added back from my leftover branch? Is this still correct?

Reply:
Initialization of these variables was moved to the header file. You should remove it and initialize `camera_2d` in the header.

Reply:
:+1: